FileShareClient/Models/BatchAttributesSearchResponse.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace UKHO.FileShareClient.Models
+{
+    [DataContract]
+    public class BatchAttributesSearchResponse : IEquatable<BatchAttributesSearchResponse>
```

Comment:
Please remove the `IEquatable` interface implementation from `BatchAttributesSearchResponse`